src/main/java/carrace/domain/Car.java

```diff
@@ -0,0 +1,46 @@
+package carrace.domain;
+
+import java.util.Random;
+
+public class Car {
+    private static final int MAX_RANGE = 9;
+    private static final int MIN_FORWARD_NUMBER = 4;
+
+    private final Random random = new Random();
+    private int position = 1;
```

Comment:
```suggestion
    private int position;
```
상수가 아니니 초기화는 기본 생성자에서 하는게 좋겠네요

Reply:
반영완료~!